frontend/src/App.tsx

```diff
@@ -18,9 +18,6 @@ const Routes: React.FunctionComponent = () => {
       <Route exact path="/">
         <Home />
       </Route>
-      <Route exact path="/report-sightings">
-        <ReportSightings />
-      </Route>
```

Comment:
This change will probably lead to a conflict, as there's another ticket about to be merged that also removes this. Don't worry though - it'll be very easy to resolve! Let me know if there's an issue 😊